src/Gateways/PaymentMethods/WCGatewayMoneiBizum.php

```
@@ -112,6 +123,39 @@ public function init_form_fields() {
 		$this->form_fields = require WC_Monei()->plugin_path() . '/includes/admin/monei-bizum-settings.php';
 	}
 
+	/**
+	 * Validate bizum_style field
+	 *
+	 * @param string $key
+	 * @param string $value
+	 * @return string
+	 */
+	public function validate_bizum_style_field( $key, $value ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)